src/test/java/org/opensearch/security/dlic/rest/api/TenantInfoActionTest.java

```diff
@@ -12,16 +12,27 @@
 package org.opensearch.security.dlic.rest.api;
 
 import org.apache.hc.core5.http.Header;
-import org.apache.hc.core5.http.HttpStatus;
+import org.apache.http.HttpStatus;
 import org.junit.Assert;
 import org.junit.Test;
 
 import org.opensearch.common.settings.Settings;
 import org.opensearch.security.support.ConfigConstants;
 import org.opensearch.security.test.helper.rest.RestHelper;
+import org.opensearch.test.framework.AsyncActions;
```

Comment:
(!) Sorry I didn't see this sooner, this package is only available to tests that are in the `integrationTest` project.  You'll need to port the new test over to that framework.